aten/src/ATen/native/TensorShape.cpp

```diff
@@ -608,5 +608,15 @@ int64_t numel(const Tensor& self) {
   return self.pImpl->numel();
 }
 
+std::vector<Tensor> unbind(const Tensor &self, int64_t dim) {
+  dim = maybe_wrap_dim(dim, self.dim());
+  int64_t size = self.size(dim);
+  std::vector<Tensor> tensors(size);
+  for (int i = 0; i < size; i++) {
+    tensors[i] = self.select(dim, i);
+  }
+  return tensors;
+}
```

Comment:
@colesbury do you remember if we need to do anything special to make outputs to be treated as views? I'm not sure how the fact that they come from `select` internally plays with the custom gradient definition

Reply:
Yes, it needs to be added to VIEW_FUNCTIONS in `gen_variable_type.py`, but that probably does not support std::vector<Tensor>

Reply:
@apaszke @colesbury I have added it to `VIEW_FUNCTIONS` and add an overload of `as_view` for vector of tensors.